Products/CMFPlone/controlpanel/browser/recyclebin.py

```diff
@@ -0,0 +1,46 @@
+# testcp.py
+from plone.app.registry.browser.controlpanel import ControlPanelFormWrapper
+from plone.app.registry.browser.controlpanel import RegistryEditForm
+from plone.z3cform import layout
+from zope import schema
+from zope.interface import Interface
+
+
+class IRecycleBinControlPanelSettings(Interface):
+    recycling_enabled = schema.Bool(
+        title="Enable the Recycle Bin",
+        description="Enable or disable the Recycle Bin feature.",
+        default=True,
```

Comment:
We should make it default to False here, but set it to True in registry.xml. That way it is enabled when a new site is created, but not enabled automatically when a site is upgraded (which might be surprising).

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4168/commits/520d440081cd9d612c436ba16b2be87eba9ea2bc